### DISCUSSION TITLE
What would a `/accessibility` page look like?

### DISCUSSION THREAD

**Discussion by oliverlloyd** [guardian/dotcom-rendering#5860] [RFCs]
If there was a dedicated https://theguardian.com/accessibility page, what would it look like?

There's a ticket to migrate the existing https://www.theguardian.com/help/accessibility-help page to DCR and we're interested in what additional content, links, features, etc. we could potentially factor in

**Comment by paperboyo** [comment]
It would contain all related controls. Every control would be a Guardian override (multiplier/on top of) of OS/browser-level controls:
– font size multiplier (goes without saying, the initial font sizes should be consistent on all our platforms)
– dark mode override
– reduced motion override

Overrides/multipliers would first read the relevant OS/browser settings and the store user settings on top. Those, in turn, could travel with a Sign in providing an incentive nicer than the gate (for not signed-in users, it should use localstorage or whatevs)

**Reply by mxdvl** [reply]
It contains an opt-in for dark mode beta as of today: #11976

**Comment by oliverlloyd** [comment]
From @SiAdcock Starter for 10: https://www.ft.com/accessibility

**Comment by ParisaTork** [comment]
[Scope](https://www.scope.org.uk/accessibility/#:~:text=Scope%27s%20commitment%20to%20website%20accessibility,of%20our%20Everyday%20Equality%20strategy.) and the [W3 website itself](https://www.w3.org/WAI/fundamentals/accessibility-intro/)

**Comment by oliverlloyd** [comment]
The NYT: https://help.nytimes.com/hc/en-us/articles/115015727108-Accessibility

**Comment by oliverlloyd** [comment]
The Washington Post: https://www.washingtonpost.com/accessibility/

**Comment by oliverlloyd** [comment]
BBC https://www.bbc.co.uk/accessibility/

**Comment by oliverlloyd** [comment]
SSE Energy https://sse.co.uk/help/accessibility
